src/test/java/com/epam/tc/hw4/steps/ActionStep.java

```diff
@@ -0,0 +1,57 @@
+package com.epam.tc.hw4.steps;
+
+import com.epam.pages.DifferentElementsPage;
+import com.epam.pages.MainPage;
+import io.qameta.allure.Step;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class ActionStep extends AbstractStep {
+
+    public ActionStep(WebDriver driver) {
+        super(driver);
+    }
+
+    @Step("Open test site by URL")
+    public void openWebSite(String website) {
+        webDriver.get(website);
```

Comment:
This method could be placed it your page object, and you may also call it openWebSite() or as you wish. And then you could say mainPage.openWebSite(<some_url>).